projects/plugins/jetpack/changelog/add-cpt-menu-calypso-menu

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
```suggestion
Significance: minor
```

I kinda feel this is a significant-enough change to call it `minor`. What do you think?

Reply:
Sure!, minor is a better significance for this change 👌